$NetBSD: patch-ab,v 1.3 2026/02/10 20:17:59 jakllsch Exp $

Beware of DOS linebreaks.

Begin to fix argv variable declaration for use with execvp().

--- execunix.c.orig	2006-06-05 10:52:27.000000000 +0000
+++ execunix.c
@@ -102,7 +102,7 @@ execcmd(
 {
 	int pid;
 	int slot;
-	const char *argv[ MAXARGC + 1 ];	/* +1 for NULL */
+	char *argv[ MAXARGC + 1 ];	/* +1 for NULL */
 
 # ifdef USE_EXECNT
 	char *p;
